README.md

```diff
@@ -1,25 +1,233 @@
+
+with cell_census.open_soma() as census:
+    adata = cell_census.get_anndata(
+        census = census,
+        organism = "Homo sapiens",
+        var_value_filter = "feature_id in ['ENSG00000161798', 'ENSG00000188229']",
+        obs_value_filter = "sex == 'female' and cell_type in ['microglial cell', 'neuron']",
+        column_names = {"obs": ["assay", "cell_type", "tissue", "tissue_general", "suspension_type", "disease"]},
+    )
+    
+    print(adata)
+    
+```
+
+The output with about 300K cells and 2 genes can be now used for downstream analysis using [scanpy](https://scanpy.readthedocs.io/en/stable/).
+
+``` bash
+AnnData object with n_obs × n_vars = 299622 × 2
+    obs: 'assay', 'cell_type', 'tissue', 'tissue_general', 'suspension_type', 'disease', 'sex'
+    var: 'soma_joinid', 'feature_id', 'feature_name', 'feature_length'
+```
+
+##### Memory-efficient queries
+
+This example provides a demonstration to access the data for larger-than-memory operations using **TileDB-SOMA** operations. 
+
+First we initiate a lazy-evaluation query to access all brain and male cells from human. This query needs to be closed — `query.close()` — or used called in a context manager — `with ...`.
+
+```python
+import cell_census
+
+with cell_census.open_soma() as census:
+    
+    human = census["census_data"]["homo_sapiens"]
+    query = human.axis_query(
+    measurement_name = "RNA",
+    obs_query = tiledbsoma.AxisQuery(
+        value_filter = "tissue == 'brain' and sex == 'male'"
+    )
+    
+    # Continued below
+
+```
+
+Now we can iterate over the matrix count, as well as the cell and gene metadata. For example, to iterate over the matrix count, we can get an iterator and perform operations for each iteration.
+
+```python
+    # Continued from above 
+    
+    iterator = query.X("raw").tables()
+    
+    # Get an iterative slice as pyarrow.Table
+    raw_slice = next (iterator) 
+    ...
+```
+
+And you can now perform operation on each iteration slice. As with any any Python iterator this logic can be wrapped around a `for` loop.
+
+And you must close the query.
+
+```
+    # Continued from above
+    query.close()
+```
+
+### R quick start
+
+*Coming soon.*
+
+
+## Questions, feedback and issues
+
+- Questions: we encourage you to ask questions via [github issues](https://github.com/chanzuckerberg/cell-census/issues). Alternatively, for quick support you can join the C[ZI Science Community](https://join-cellxgene-users.herokuapp.com/) on Slack and join the `#cell-census-users` channel
+- Bugs: please submit a [github issue](https://github.com/chanzuckerberg/cell-census/issues). 
+- Feature requests: please submit requests using this [form](https://airtable.com/shrVV1g0d6nvBoQYu).
+- Security issue: if you believe you have found a security issue, we would appreciate notification. Please send an email to <security@chanzuckerberg.com>.
+- You can send any other feedback to <soma@chanzuckerberg.com>
+
+
+## Coming soon
+
+- R support!
```

Comment:
Could be valuable to plunk these as "Feature request" seed in github discussions and see what the community thinks...